### PR TITLE
Add basil, minor code fixes

### PR DESCRIPTION
**Pull request by placuszek15**
Hi, I've added in-game basil's sprites from the console version of OMORI, also updated the omori sprite to contain the unused afraid sprite and made the crop utility a bit better (no idea how you were supposed to be using it beforehand, now you can just ``node crop.mjs`` and get all the thumbnails)

### CODE REVIEW

**Comment by gBasil** [comment]
Hey there! At first glance, it looks good to me, though I'll request one tiny change. I'm currently down with COVID-19, so I'll merge this PR and make some changes to bring the repo up to date with my current environment (i.e. npm to Yarn, moving sharp to `devDependencies`) once I feel better.

Thank you so much!

File: crop.mjs [hunk 1]
```diff
@@ -48,37 +48,59 @@ const lineNames = {
...
+					count: lineNames.sunny.length,
 					names: lineNames.sunny,
 			  }
 			: file === 'omori.png'
 			? {
-					count: 15,
+					count: lineNames.omori.length,
 					names: lineNames.omori,
 			  }
+			: file == "basil.png"
```
Comment (gBasil) [inline crop.mjs]: Could you please change this from `"basil.png"` to `'basil.png'`?

**Review by gBasil — CHANGES_REQUESTED** [review]

File: crop.mjs (hunk 1 above)
Comment (placuszek15) [reply crop.mjs]: Done!

**Review by gBasil — APPROVED** [review]

**Comment by gBasil** [comment]
Thank you, sorry it took so long!

**Comment by Basil-Omori3** [comment]
can you add faraway town Basil?

**Pull request merged**